### ISSUE TITLE
Is there any tool to compute map?

### ISSUE THREAD

**Issue by banjuanshua** [TencentYoutuResearch/ObjectDetection-OneStageDet#6]

**Comment by mileistone** [comment]
you can find that tool in pascal voc website or just refer to [this](https://github.com/rbgirshick/py-faster-rcnn/blob/master/lib/datasets/voc_eval.py)

**Comment by banjuanshua** [comment]
Thank you. If a custom dataloader will you provide? I think now dataloader in it is a little hard to use.

**Comment by banjuanshua** [comment]
Your dataloader needs bbb.parse(anno_format, anno_filename, identify=lambda f: f, class_label_map=class_label_map, **kwargs) to construct annos, so a custom dataloader needs anon_format first. I am confuesd at yolov3 loss in Pytorch all the time,  however, thanks for your repo.

**Comment by muye5** [comment]
@mileistone which way should the map of voc be caculated in, use_07_metric is False or True for your results (Whether to use VOC07's 11 point AP computation), thank you.

**Comment by yanxp** [comment]
hello, why the mAP of voc2007 is so low only 11.2 when i use the code?

**Comment by GUOShuxuan** [comment]
Hi, I am new in detection. Could you mind release the map_eval script without readme? I am trying to test the results of some new model.

**Comment by wangg12** [comment]
@mileistone Could you kindly add a map_eval script?

**Comment by wangg12** [comment]
@mileistone I have written a map_eval script but can only get 50.7 with the model provided by you. Could you release your eval script?
